.github/workflows/odh-notebooks-sync.yml

```diff
@@ -0,0 +1,114 @@
+# The aim of this GitHub workflow is to update the pipfile to sync with Codeflare-SDK release.
+name: Sync with codeflare-sdk release
+on:
+  workflow_dispatch:
+    inputs:
+      upstream-repository-organization:
+        required: true
+        description: "Provide the upstream repository organization name"
+
+      codeflare-repository-organization:
+        required: true
+        description: "Provide the Codeflare repository organization name"
+
+      codeflare_sdk_release_version:
+        required: true
+        description: "Provide version of the Codeflare-SDK release"
```

Comment:
Is there a reason why it is necessary to gather the upstream/codeflare repos from a user if its just Project CodeFlare and ODH we are syncing to?